v0/destinations/marketo/transform.js

```diff
@@ -127,17 +141,20 @@ const createOrUpdateLead = async (
 const lookupLeadUsingEmail = async (formattedDestination, token, email) => {
   return emailLeadCache.get(email, async () => {
     stats.increment(LEAD_LOOKUP_METRIC, 1, { type: "email", action: "fetch" });
-    const resp = await sendGetRequest(
-      `https://${formattedDestination.accountId}.mktorest.com/rest/v1/leads.json`,
-      // `https://httpstat.us/200`,
-      {
-        params: { filterValues: email, filterType: "email" },
-        headers: { Authorization: `Bearer ${token}` }
-      },
-      "During lead look up using email"
-    );
-    if (resp) {
-      const { result } = resp;
+    const response = marketoResponseHandler({
+      clientResponse: await sendGetRequest(
```

Comment:
Similarly we are not using `httpGET` method here ?

Reply:
Just like post to we are using a wrapper on top of httpGET for marketo which is compatible with responseHandler